haas/api.py

```diff
@@ -560,21 +565,28 @@ def headnode_detach_network(headnode, hnic):
 
 @rest_call('GET', '/networks')
 def list_networks():
+    """Lists all networks
+
+    """
     get_auth_backend().require_admin()
 
     networks = db.session.query(model.Network).all()
     result = {}
     for n in networks:
         if n.access:
-            net = {'driver_id': n.network_id, 'projects': [p.label for p in n.access]}
+            net = {'network_id': n.network_id, 'projects': [p.label for p in n.access]}
         else:
-            net = {'driver_id': n.network_id, 'projects': None}
+            net = {'network_id': n.network_id, 'projects': None}
         result[n.label] = net
             
     return json.dumps(result, sort_keys = True)
 
 @rest_call('GET', '/network/<network>/attachments')
 def list_network_attachments(network, project=None):
+    """Lists all the attachments from <project> for <network>
+
+    If <project> is not specified, lists all attachments for <network>
+    """
```

Comment:
If you're not going to have anything else in the docstring, you should have the closing quote on the same line.